keyboards/herevoland/buff75/matrix_diagram.md

```diff
@@ -0,0 +1,18 @@
+# Matrix Diagram for Buff75
+
+```
+            ┌───┐ ┌───┬───┬───┬───┐ ┌───┬───┬───┬───┐ ┌───┬───┬───┬───┐ ┌───┐
+            │00 │ │01 │02 │03 │04 │ │06 │07 │08 │09 │ │0A │0B │0C │0D │ │0E │
+            └───┘ └───┴───┴───┴───┘ └───┴───┴───┴───┘ └───┴───┴───┴───┘ └───┘
+            ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
```

Comment:
Your product image, this matrix diagram, keymaps, and keyboard.json still do not line up.

Image shows a 2u backspace, while this suggests split backspace.

Comparing the 2nd row
* image.jpg - 15 keys
* keymap.c 2nd row - 15 keys
* matrix_diagram.md - 16 keys
* keyboard.json - 16 keys